### PR TITLE
Fix plugin dev and kuzzle images

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do ?

This PR adapt the `kuzzleio/plugin-dev` and `kuzzleio/kuzzle` images to be build manually in travis.   

The `kuzzleio/plugin-dev` image will download Kuzzle source code from Github. You can specify the branch with a build argument (eg: `--build-arg kuzzle_branch=1.x`)

The `kuzzleio/kuzzle` image is based on the `kuzzleio/plugin-dev` image with a multi stage build. You can specify the tag of the `kuzzleio/plugin-dev` image that should be used to build `kuzzleio/kuzzle` (eg: `--build-arg current_tag=develop`)

### How should this be manually tested?

  - Step 1 : Build `kuzzleio/plugin-dev` with `1.x` branch : `docker build -t kuzzleio/plugin-dev:develop -f plugin-dev/Dockerfile --build-arg kuzzle_branch=1.x plugin-dev`
  - Step 2 : Build `kuzzleio/kuzzle` with `kuzzleio/plugin-dev` on tag `develop`: `docker build -t kuzzleio/kuzzle:develop -f kuzzle/Dockerfile --build-arg current_tag=develop kuzzle`
  - Step 3 :  Each image should contain the same Kuzzle version as on the `1.x` branch.

### CODE REVIEW

File: plugin-dev/README.md [hunk 1]
```diff
@@ -24,5 +24,5 @@ Contains:
 
 Theses images are build using multi-stage Dockerfile.
 
-The first stage (`development`) is the development image which install packages, grab the source code and build npm packages.  
-The second stage (`production`) only copy the application directory and nodejs binaries inside the container.  
+The first stage (`plugin-dev`) is the development image which install packages, grab the source code and build npm packages.
```
Comment (scottinet) [inline plugin-dev/README.md]: install => installs
grab => grabs
build => builds

File: plugin-dev/README.md [hunk 2]
```diff
@@ -24,5 +24,5 @@ Contains:
 
 Theses images are build using multi-stage Dockerfile.
 
-The first stage (`development`) is the development image which install packages, grab the source code and build npm packages.  
-The second stage (`production`) only copy the application directory and nodejs binaries inside the container.  
+The first stage (`plugin-dev`) is the development image which install packages, grab the source code and build npm packages.  
+The second stage (`kuzzle`) only copy the application directory and nodejs binaries inside the container.
```
Comment (scottinet) [inline plugin-dev/README.md]: copy => copies

File: plugin-dev/README.md [hunk 3]
```diff
@@ -24,5 +24,5 @@ Contains:
 
 Theses images are build using multi-stage Dockerfile.
 
-The first stage (`development`) is the development image which install packages, grab the source code and build npm packages.  
-The second stage (`production`) only copy the application directory and nodejs binaries inside the container.  
+The first stage (`plugin-dev`) is the development image which installs packages, grabs the source code and builds npm packages.
```
Comment (ballinette) [inline plugin-dev/README.md]: I don't notice where there are 2 stages in Dockerfile for `plugin-dev` ?

Comment (Aschen) [reply plugin-dev/README.md]: There is not a second stage directly in the `kuzzleio/plugin-dev` Dockerfile but the builded image is used in a multi-stage build for `kuzzleio/kuzzle`

**Pull request merged**